projects/plugins/jetpack/extensions/blocks/conversation/example.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+
+ import { __ } from '@wordpress/i18n';
+
+const participants = [
+	{
+		participantSlug: 'participant-0',
+		participant: 'Rosalind',
+		hasBoldStyle: true,
+		hasUppercaseStyle: true,
+	},
+	{
+		participantSlug: 'participant-1',
+		participant: 'Orlando',
```

Comment:
```suggestion
		participant: __( 'Orlando', 'jetpack' ),
```

Reply:
same. I think we shouldn't translate the names.